components/mail/dovecot-pigeonhole/pigeonhole.p5m

```diff
@@ -0,0 +1,130 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Josef 'Jeff' Sipek <jeffpc@josefsipek.net>
+#
+
+<transform file path=usr/libexec/dovecot/.* -> default mode 0555>
+<transform file path=.* -> default pkg.depend.bypass-generate libdovecot(|-storage|-lda|-login)\.so\.0>
```

Comment:
On Thu, Sep 08, 2016 at 20:41:24 -0700, Alexander Pyhalov wrote:
> > +# This file and its contents are supplied under the terms of the
> > +# Common Development and Distribution License ("CDDL"), version 1.0.
> > +# You may only use this file in accordance with the terms of version
> > +# 1.0 of the CDDL.
> > +#
> > +# A full copy of the text of the CDDL should have accompanied this
> > +# source.  A copy of the CDDL is also available via the Internet at
> > +# http://www.illumos.org/license/CDDL.
> > +#
> > +
> > +#
> > +# Copyright 2016 Josef 'Jeff' Sipek <jeffpc@josefsipek.net>
> > +#
> > +
> > +<transform file path=usr/libexec/dovecot/.* -> default mode 0555>
> > +<transform file path=.* -> default pkg.depend.bypass-generate libdovecot(|-storage|-lda|-login)\.so\.0>
> 
> Why do we need this? Perhaps, we have to add some -R flags?

libdovecot*.so lives in a non-standard path (usr/lib/dovecot/).  It's just
pkgdepend that's confused.  There are no issues at runtime, since the
dovecot executables contain:

       [9]  RUNPATH           0x4194              /usr/lib/dovecot
      [10]  RPATH             0x4194              /usr/lib/dovecot

I guess we could add -R when building external modules instead of shutting up
pkgdepend.

-- 
Debugging is twice as hard as writing the code in the first place.
Therefore, if you write the code as cleverly as possible, you are, by
definition, not smart enough to debug it.
		- Brian W. Kernighan